### PR TITLE
Add api plugin

### PR DESCRIPTION
**Pull request by nightink**
refactor: module
- refactor index.js
- add test
- #1

### CODE REVIEW

**Comment by nightink** [comment]
@elrrrrrrr ping

**Comment by elrrrrrrr** [comment]
+1

File: bin/nff [hunk 1]
```diff
@@ -15,4 +49,28 @@ if(!program.find) {
...
+  ignores: program.ignore && program.ignore.split(','),
+  findKeys: program.find.split(','),
+  wherePaths: program.where && program.where.split(','),
+  cwdPath: process.cwd()
+};
+
+console.log('\n\n========== node.js find {%s} list ==========\n\n', options.findKeys);
+
+require('..')(options, function(err, findWords) {
+  for(var findKey in findWords) {
```
Comment (elrrrrrrr) [inline bin/nff]: 考虑下改成forEach呗

Comment (nightink) [inline bin/nff]: 这里是对象，用 Object.keys 哈哈

**Pull request merged**